static/swagger-specs/destinations.yaml

```diff
@@ -3313,10 +3313,21 @@ components:
         - interval
         - timeUnit
       properties:
+        exportMode:
+          title: Export mode
+          type: string
+          description: Select `"DAILY_FULL_EXPORT"` or `"FIRST_FULL_THEN_INCREMENTAL"`. `"FIRST_FULL_THEN_INCREMENTAL"` triggers an export of one or multiple files representing a full snapshot of the dataset. Subsequent files are incremental additions to the dataset since the previous export.
```

Comment:
Minor nit - I don't think we need this in both `` and ""? We can probably just keep it to one of them.

Additionally, since we're explaining what "FIRST_FULL_THEN_INCREMENTAL" is, we should probably also explain what "DAILY_FULL_EXPORT" does for completion's sake.